src/gsi/stppm2_5.f90

```diff
@@ -161,6 +166,100 @@ subroutine stppm2_5(pm2_5head,rval,sval,out,sges,nstep)
 
     endif
 
+    if ( fv3_cmaq_regional .and. laeroana_fv3cmaq) then
+       pm2_5ptr => pm2_5Node_typecast(pm2_5head)
+
+       do while (associated(pm2_5ptr))
+          if(pm2_5ptr%luse)then
+             if(nstep > 0)then
+                j1=pm2_5ptr%ij(1)
+                j2=pm2_5ptr%ij(2)
+                j3=pm2_5ptr%ij(3)
+                j4=pm2_5ptr%ij(4)
+                j5=pm2_5ptr%ij(5)
+                j6=pm2_5ptr%ij(6)
+                j7=pm2_5ptr%ij(7)
+                j8=pm2_5ptr%ij(8)
+                w1=pm2_5ptr%wij(1)
+                w2=pm2_5ptr%wij(2)
+                w3=pm2_5ptr%wij(3)
+                w4=pm2_5ptr%wij(4)
+                w5=pm2_5ptr%wij(5)
+                w6=pm2_5ptr%wij(6)
+                w7=pm2_5ptr%wij(7)
+                w8=pm2_5ptr%wij(8)
+
+                istatus=0
+
+                val2=-pm2_5ptr%res
+                val=zero
+
+                !aeroname='aso4i'
```

Comment:
Remove this, if it isn't needed at all/eventually, if it is, explain why it is commented out.